packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -291,13 +299,13 @@ class TextField extends React.Component<PropsWithForwardRef, State> {
                         placeholder={placeholder}
                         value={value}
                         name={name}
-                        disabled={disabled}
+                        aria-disabled={disabled}
                         onChange={this.handleChange}
-                        onKeyDown={onKeyDown}
-                        onFocus={this.handleFocus}
-                        onBlur={this.handleBlur}
+                        onKeyDown={disabled ? undefined : onKeyDown}
+                        onFocus={this.handleFocus} // TextField can be focused if disabled
+                        onBlur={this.handleBlur} // TextField can be blurred if disabled
                         data-testid={testId}
-                        readOnly={readOnly}
+                        readOnly={readOnly || disabled} // Set readOnly also if it is disabled, otherwise users can type in the field
```

Comment:
**side note:** It's great that you are including this here so the `Combobox` gets this by free :)